bin/ci-annotate-errors

```diff
@@ -12,4 +12,9 @@
 # ci-annotate-errors - Detect errors in junit xml as well as log files during
 # CI and find associated open GitHub issues in Materialize repository.
 
-exec "$(dirname "$0")"/pyactivate -m materialize.cli.ci_annotate_errors "$@"
+HOSTNAME=$(exec "$(dirname "$0")"/ci-builder run stable "$(dirname "$0")"/mzcompose --mz-quiet --find get-cloud-hostname run default --app-password "$PRODUCTION_ANALYTICS_APP_PASSWORD")
```

Comment:
I don't think the app password should be passed as a command line flag, I'd prefer to pass through PRODUCTION_ANALYTICS_APP_PASSWORD in bin/ci-builder as an allow-listed environment variable

Reply:
I understand and am also not that happy about it. Though, your suggestion reduces some flexibility by kind of hardcoding the chosen password and thereby the chosen environment.
I think I will pick up your suggestion but provide the environment variable name as input argument to allow other choices as well.

Reply:
=> https://github.com/MaterializeInc/materialize/pull/27514/commits/1e622d73fb7232ad7e0fece17a497ec550973df7 and adjusted `bin/ci-annotate-errors` in an amended commit.